pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -0,0 +1,629 @@
+// api.pb.go could be generate by hack/update-generate-runtime.sh
+syntax = 'proto2';
```

Comment:
Choose proto2 instead of proto3 because

- Kubernetes already have a lot of proto definitions, they are all based on proto2
- Kubernetes use protoc-gen-gogo for protobuf generation, but proto3 is not fully supported now, see https://github.com/gogo/protobuf/blob/master/Readme.md#proto3